book/src/user_manual/processing.md

```diff
@@ -45,6 +45,15 @@ Standalone proposals are returned as a `QueuedProposal`, indicating that they ar
 {{#include ../../../openmls/tests/book_code.rs:inspect_add_proposal}}
 ```
 
+#### Rolling back proposals
+
+Operations that add a proposal to the queue, will return its reference. This reference can be used to remove
+a proposal from the store. This can be useful for example to rollback in case of errors.
```

Comment:
```suggestion
Operations that add a proposal to the proposal store, will return its reference. This reference can be used to remove
a proposal from the proposal store. This can be useful for example to roll back in case of errors.
```